### ISSUE TITLE
Add sample CORS filter to faciliate iterative front-end development

### ISSUE THREAD

**Issue by scottkurz** [WASdev/sample.batch.tweetanalyzer#3]
In dev we added a CORS filter enabling various requests from the front-end Javascript code to the back-end REST API.   Thinking it'd be a bit tighter to remove it we did, but we should include it in commented out form for someone else's development.

**Comment by scottkurz** [comment]
The filter looked like this originally, at path **src/main/java/com/ibm/websphere/sample/fetcher/CorsFilter.java**:
```
-package com.ibm.websphere.sample.fetcher;
-
-import java.io.IOException;
-
-import javax.ws.rs.container.ContainerRequestContext;
-import javax.ws.rs.container.ContainerResponseContext;
-import javax.ws.rs.container.ContainerResponseFilter;
-import javax.ws.rs.ext.Provider;
-
-@Provider
-public class CorsFilter implements ContainerResponseFilter {
-
-   @Override
-   public void filter(final ContainerRequestContext requestContext,
-                      final ContainerResponseContext cres) throws IOException {
-      cres.getHeaders().add("Access-Control-Allow-Origin", "*");
-      cres.getHeaders().add("Access-Control-Allow-Headers", "origin, content-type, accept, authorization");
-      cres.getHeaders().add("Access-Control-Allow-Credentials", "true");
-      cres.getHeaders().add("Access-Control-Allow-Methods", "GET, POST, PUT, DELETE, OPTIONS, HEAD");
-      cres.getHeaders().add("Access-Control-Max-Age", "1209600");
-   }
-
-}
```

**Comment by scottkurz** [comment]
Resolved in commit 99c017d
